vm/src/function.rs

```diff
@@ -401,6 +401,13 @@ impl<T> OptionalArg<T> {
             Missing => default(),
         }
     }
+
+    pub fn opt_ref(&self) -> Option<&T> {
```

Comment:
maybe `as_opt_ref` is closer to rust naming convention?

Reply:
I removed the function, I originally tried to implement `_codecs` in Rust and I was using it a lot more there, but now it was just 1 use so I just took it out.